bin/add-extension-package.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
```

Comment:
Since this script can be used to add all kinds of packages (`extensions`, `packages`, `acceptance`, etc.), I find the name `add-extension-package.js` as misleading. Let's find a better one, for example:

- `bin/create-package.js`
- `bin/add-new-package.js`